solutions/swb-reference/integration-tests/tests/isolated/projects/create.test.ts

```diff
@@ -0,0 +1,164 @@
+/*
+ *  Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *  SPDX-License-Identifier: Apache-2.0
+ */
+
+import ClientSession from '../../../support/clientSession';
+import Setup from '../../../support/setup';
+import HttpError from '../../../support/utils/HttpError';
+import { checkHttpError } from '../../../support/utils/utilities';
+
+interface CreateRequest {
+  name?: string;
+  description?: string;
+  costCenterId?: string;
+}
+
+describe('Create Project negative tests', () => {
+  const setup: Setup = new Setup();
+  let adminSession: ClientSession;
+  const costCenterId = setup.getSettings().get('costCenterId');
+
+  beforeEach(() => {
+    expect.hasAssertions();
+  });
+
+  beforeAll(async () => {
+    adminSession = await setup.getDefaultAdminSession();
+  });
+
+  afterAll(async () => {
+    await setup.cleanup();
+  });
+
+  describe('with invalid name values', () => {
+    const createRequest: CreateRequest = {
+      description: 'new project who dis',
+      costCenterId
+    };
+
+    describe('with name that belongs to an existing project', () => {
+      const existingProjectName = setup.getSettings().get('projectName');
```

Comment:
as we already have a default projectId, we could also just retrieve the project by id and the name from that

Reply:
I added instructions in the readme for setting up integ tests to provide the project name in the config that corresponds to the projectId already provided. The problem I'd have with retrieving by projectId is that it would not longer be an isolated test since it is testing get and create.